CHANGELOG.md

```diff
@@ -38,7 +38,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed a problem where the "editor" information has been duplicated during saving a .bib-Database. [#5359](https://github.com/JabRef/jabref/issues/5359)
 - We re-introduced the feature to switch between different preview styles. [#5221](https://github.com/JabRef/jabref/issues/5221)
 - We fixed various issues (including [#5263](https://github.com/JabRef/jabref/issues/5263)) related to copying entries to the clipboard
-
+- We added a change listener to the main table to scroll to imported entry [#5383](https://github.com/JabRef/jabref/issues/5383)
```

Comment:
The change log is mostly for our users and most of them don't know what a "change listener" is ;-). Please reformulate this a bit more user-friendly.

Reply:
Done.